src/utils/Client.ts

```diff
@@ -71,31 +81,81 @@ export class Client extends Discord.Client {
 
 	public async addSlashCommands(path: string): Promise<this> {
 		try {
+			// eslint-disable-next-line no-await-in-loop
+			while (!this.user) await this.readyPromise;

```

Comment:
You didn't understand anything lol
```suggestion
			await this.readyPromise;
```

Reply:
I didn't

Reply:
I noticed

Reply:
Thanks :heart: